instrumentation/opentelemetry/errorhandler/handler.go

```diff
@@ -0,0 +1,22 @@
+package errorhandler
```

Comment:
Can you check if this works for our custom batchspanprocessor or if it can be adapted the new zap logger? https://github.com/hypertrace/goagent/tree/main/instrumentation/opentelemetry/batchspanprocessor . We add some custom code in our batchspanprocessor like catching panics and metrics for spans received, sent and dropped. Since we cannot use the logger in the internal package in the otel go repo, we adapt it and even initialize a new stdout logger.

Reply:
I didnt know we had a logger here. We can do it, zap has a global api. We can either wrap it in go-logr or use zap itself in the custom bsp

Reply:
done